test/CommonTests.hs

```diff
@@ -49,6 +51,44 @@ tests =
        -- Because excel treats 1900 as a leap year, dateToNumber and dateFromNumber
        -- aren't inverses of each other in the range n E [60, 61[ for DateBase1900
        \b (n :: Pico) -> (n >= 60 && n < 61 && b == DateBase1900) || n == dateToNumber b (dateFromNumber b $ n)
+    , testCase "building single CellRefs" $ do
```

Comment:
what do you think about adding some property tests?

Reply:
I'll look into it.

I'll probably end up describing a competing definition of cellref just to prop'test the builders/parsers with Serial :)

Reply:
I've added matching property tests around producing and parsing CellRefs.

I've also realised I didn't add support for "foreign" single cell refs so I've added that too